tests/integration/app_mock.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Comment:
I could get rid of the python app mock. Now re-using the existing one, rewriting the classpath for the container environment and mounting the target directories.